documentation/modules/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -7,7 +7,31 @@
 
 The following resources will created by the Cluster Operator in the Kubernetes cluster:
 
+.Shared resources
+
+`_cluster-name_-cluster-ca`:: Secret with the Cluster CA used to encrypt the cluster communication.
+`_cluster-name_-cluster-ca-cert`:: Secret with the Cluster CA public key. This key can be used to verify the identity of the Kafka brokers.
+`_cluster-name_-clients-ca`::  Secret with the Clients CA used to encrypt the communication between Kafka brokers and Kafka clients.
+`_cluster-name_-clients-ca-cert`:: Secret with the Clients CA public key. This key can be used to verify the identity of the Kafka brokers.
```

Comment:
It's used to sign the client's private key so it's used to verify the identity of Kafka clients (during mutual authentication), or?